scripts/expo/commands/publishCodeReview.js

```diff
@@ -0,0 +1,87 @@
+const fetch = require('node-fetch');
+
+const spawn = require('../spawn');
+const utils = require('../utils');
+
+/**
+ * TRAVIS_PULL_REQUEST_BRANCH
+ *   - if the current job is a pull request, the name of the branch from which the PR originated.
+ *   - if the current job is a push build, this variable is empty ("").
+ *
+ * TRAVIS_BRANCH
+ *   - for push builds, or builds not triggered by a pull request, this is the name of the branch.
+ *   - for builds triggered by a pull request this is the name of the branch targeted by the pull request.
+ *   - for builds triggered by a tag, this is the same as the name of the tag (TRAVIS_TAG).
+ *
+ * @see https://docs.travis-ci.com/user/environment-variables/
+ */
+function preparePublishName() {
+  const branchName =
+    process.env.TRAVIS_PULL_REQUEST_BRANCH || process.env.TRAVIS_BRANCH;
+  return `code-review-${branchName}`.replace(/[^a-zA-Z0-9\\-]/, '-');
+}
+
+module.exports = function publishCodeReview() {
+  const config = utils.getExpoConfig();
+  const slug = preparePublishName();
+  utils.saveExpoConfig({
+    expo: Object.assign(config.expo, {
+      slug: slug,
+      name: slug,
+      privacy: 'unlisted',
+      packagerOpts: {
+        nonPersistent: true,
+      },
+    }),
+  });
+
+  const githubPullRequestId = process.env.TRAVIS_PULL_REQUEST;
+  spawn('yarn', ['exp', 'publish'], publishError => {
+    if (publishError) {
+      throw new Error(publishError);
+    } else if (githubPullRequestId > 0) {
+      // comments only when the PR is opened (but publish always)
+      commentOnGitHub(slug, githubPullRequestId);
+    }
+  });
+};
+
+function commentOnGitHub(buildName, githubPullRequestId) {
```

Comment:
I would move fn definition above the exported function, one does not simply find exported function in the middle of file